docs/design/3_1_1_video_capture.tex

```diff
@@ -1 +1,29 @@
-TODO: Talk about camera choice, MIPI interface, camera libraries. Also, talk about alternatives we explored (i.e. analog video transmission/acquisition.
\ No newline at end of file
+% TODO: Talk about camera choice, MIPI interface, camera libraries. Also, talk about alternatives we explored (i.e. analog video transmission/acquisition.
+
+The video capture subsystem of the computing platform consists of a camera, its video transmission protocol to the processing core and video transmission to the base station. For this subsystem, the hardware used includes a CMOS camera, a Rasberry Pi device, and MIPI Camera Serial Interface protocol. Design description of the Rasberry Pi device is described in the next section.
+
+\subsubsection{Camera Choice}
+The camera used to capture video is the Rasberry Pi Camera Module. It supports 1080p, 720p and 640x480p video. It has a fixed focus lens. The camera will capture live video and send the video data to the Rasberry Pi, which will feed the data in different forms to both the machine learning model and the base station.
+
+% camera Choice
+The decision of the camera was influenced by the requirement of future replacability. The project's goal is to create a system which provides both video acquisition and supplemental programmable hardware processing capabilities. The Rasberry Pi device contains different ports that will support different cameras with different performance.
+
+One of the limitations on the resolution and framerate of the camera is the latency of the machine learning implementation on the FPGA. The machine learning implementation is not the focus of this project and will not be optimized for performance and thus it will not be necessary to use a camera with high resoltuion and framerate.
```

Comment:
```suggestion
One of the limitations on the resolution and framerate of the camera is the latency of the machine learning implementation on the FPGA. The machine learning implementation is not the focus of this project and will not be optimized for performance and thus it will not be necessary to use a camera with high resolution and framerate.
```
This sounds like the limitation on the throughput of the computing platform, not the camera.